internal/eval/hmap.go

```diff
@@ -83,3 +84,21 @@ func getValueFromHashMap(key, field string, store *dstore.Store) (val, err []byt
 
 	return clientio.Encode(value, false), nil
 }
+
+func (h HashMap) incrementValue(field string, increment int64) (int64, error) {
+	val, ok := h[field]
+	if !ok {
+		h[field] = fmt.Sprintf("%v", increment)
+		return increment, nil
+	}
+
+	i, err := strconv.ParseInt(val, 10, 64)
+	if err != nil {
+		return -1, diceerrors.NewErr(diceerrors.HashValueNotIntegerErr)
+	}
+
+	total := i + increment
```

Comment:
Another suggestion here would be to check overflow after incrementing. Please check the following scenario:
![image](https://github.com/user-attachments/assets/71f9325b-f4de-43d9-aaa8-3146ca3f913b)

Reply:
My Bad. I missed this edgecase. Thank you for pointing it out.